lib/opensusebasetest.pm

```diff
@@ -556,13 +557,57 @@ sub remount_tmp_if_ro {
     script_run 'touch /tmp/test_ro || mount -t tmpfs /dev/shm /tmp';
 }
 
+=head2 select_serial_terminal
+
+    select_serial_terminal($root);
+
+Select most suitable text console with root user.
+The choice is made by BACKEND and other variables.
+
+Optional C<root> parameter specifies, whether use root user
+(1 or default) or prefer non-root user if available.
+=cut
+sub select_serial_terminal {
+    my ($self, $root) = @_;
+    $root = shift // 1;
```

Comment:
I think `shift` will retrieve `$self` here. I think you mean this: `$root //= 1;`

Reply:
@cfconrad fixed, thanks! Yes. Original code procedure wasn't in class, so `$self` wasn't needed.